projects/validation-messages/src/lib/components/validation-messages/validation-messages.component.ts

```diff
@@ -38,64 +41,51 @@ type ApiErrorMessages =
   templateUrl: './validation-messages.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ValidationMessagesComponent
-  implements OnInit, OnDestroy, DoCheck, AfterContentInit
-{
-  materialErrorMatcher = false;
-  shownErrors: string[] = [];
-
+export class ValidationMessagesComponent implements OnInit, AfterContentInit {
   @Input() errorsMessages: ValidationMessagesConfig = {};
   @Input() control!: FormControl;
   @Input() controlName!: string;
```

Comment:
shouldn't they be optional? just wondering

Reply:
you are right, they should